### DISCUSSION TITLE
multiple Pull requests possible?

### DISCUSSION THREAD

**Discussion by josephedwardchang** [google/digitalbuildings#423] [Q&A] [answered]
Hi All, 

I already have a pull request under review #386, and I want to add another pull request in  parallel for another device we have to add: Air Ionizer. I used a github fork for the pull request. However, creating a new pull request goes to comparison of branches, where the create new pull request button is disabled. I also can't fork the ontology again for me to create a new pull request.

is multiple pull request possible?

if not, can I join the Air Ionizer changes into the current pull request #386?

or do I wait to get #386 done first before I make a new pull request?

**Comment by charbull** [comment]
Hi @josephedwardchang sorry we missed the discussions here, as they were experimental in github we didnt watch them closely.

Are you all set?
Thank you

**Comment by charbull** [comment, marked as answer]
In general you can submit multiple pull requests. Ideally the smaller the scope the faster the review and approval.
